sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.Metrics.cs

```diff
@@ -87,6 +87,10 @@ public MonitoringDataPoint GetDataPoint()
                     ApplyFilters(metricAccumulators, _collectionConfiguration.ExceptionMetrics, exception, out filteringErrors, ref projectionError);
                     liveMetricsBuffer.RecordException();
                 }
+                else if (item is Models.Trace trace)
+                {
+                    // TODO: APPLY FILTERS
```

Comment:
Need to follow up with Sean. 
Propose leaving this as TODO for this PR.

Reply:
I've added this check. You can find it in the recently merged PR: https://github.com/Azure/azure-sdk-for-net/pull/42855/files#diff-27858578e4ef519f56f03d0e3adff7d3ce3ea8e9f46d1024514915e093cd4a98